src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8744,9 +9589,6 @@ class C2 : I<string>
                 // (13,22): error CS8987: The 'scoped' modifier of parameter 'r' doesn't match overridden or implemented member.
                 //     public R<string> F2(R<string> r) => default; // 2
                 Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F2").WithArguments("r").WithLocation(13, 22),
-                // (14,22): error CS8987: The 'scoped' modifier of parameter 'r' doesn't match overridden or implemented member.
-                //     public R<string> F3(scoped ref R<string> r) => default; // 3
-                Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F3").WithArguments("r").WithLocation(14, 22),
```

Comment:
This doens't seem to match the behavior for the indexer parameter?

Reply:
`F3()` and `F4()` have `ref` parameters to `ref struct` types which is implicitly scoped, while the two indexer parameters are `ref struct` types passed by value. I've added indexer cases with `in` parameters to `ref struct` types to fill the gap.